patches/components-vector_icons-aggregate_vector_icons.py.patch

```diff
@@ -1,8 +1,28 @@
 diff --git a/components/vector_icons/aggregate_vector_icons.py b/components/vector_icons/aggregate_vector_icons.py
-index 3e5a94a7f053ffc66f1a820073265cbfe1332b39..0175ec714b441df5e8fa58e9391e2de6bbd67755 100644
+index 3e5a94a7f053f..47a09b09313d2 100644
 --- a/components/vector_icons/aggregate_vector_icons.py
 +++ b/components/vector_icons/aggregate_vector_icons.py
-@@ -102,7 +102,7 @@ def ExtractIconReps(icon_file_name):
+@@ -10,6 +10,19 @@ import re
+ import shlex
+ import sys
+ import textwrap
++import json
++from pathlib import Path
++
++def get_overrides():
++  brave_root = f'{Path(__file__).parent}/../../brave'
++  overrides_file_path =  f'{brave_root}/vector_icons/leo_overrides.json'
++
++  leo_overrides = {}
++  data = json.load(open(overrides_file_path))
++
++  for icon, leo_override in data.items():
++    leo_overrides[icon] = f'../../brave/node_modules/@brave/leo/icons-skia/{leo_override}.icon'
++  return leo_overrides
```

Comment:
For python patching we typically externalize stuff like this into a separate python file (under brave/scripts) and then try to minimize the patch using an import. For example, [script](https://github.com/brave/brave-core/blob/master/script/signing_helper.py) and [patch](https://github.com/brave/brave-core/blob/master/patches/chrome-installer-mac-signing-config_factory.py.patch)